src/wp-content/themes/twentynineteen/inc/helper-functions.php

```diff
@@ -11,7 +11,13 @@
  * Determines if post thumbnail can be displayed.
  */
 function twentynineteen_can_show_post_thumbnail() {
-	return apply_filters( 'twentynineteen_can_show_post_thumbnail', ! post_password_required() && ! is_attachment() && has_post_thumbnail() );
+	$show_post_thumbnail = ( ! post_password_required() && ! is_attachment() && has_post_thumbnail() );
+	/**
```

Comment:
The original `return` line technically could remain unedited and use the `$show_post_thumbnail` _only_ in the documentation, like [wp_is_large_user_count](https://github.com/WordPress/wordpress-develop/blob/6.8.1/src/wp-includes/user.php#L1577) does. However, the way you rewrote it with the conditions in a variable is easier to understand than having all three conditions inside `apply_filters()`.

I think I'll just ask for an empty line between assigning the variable and starting the docblock.

```suggestion

	/**
```